.github/workflows/nightly-tests.yml

```diff
@@ -32,6 +34,7 @@ jobs:
           start-xvfb: ${{ env.DISPLAY }}
       - &build
         name: Build
+        id: build
```

Comment:
I forgot to include this id when making an update in the last PR. My editor only started showing me the warning when trying to do the merge conflict resolution.